dodola/tests/test_services.py

```diff
@@ -74,6 +74,11 @@ def _gcmfactory(x, start_time="1950-01-01"):
             "time_bnds": (["time", "bnds"], np.ones((len(x), 2))),
         },
     )
+
+    if gcm_variable == "pr":
+        # assign units to typical GCM pr units so they can be cleaned
+        out["pr"].attrs["units"] = "kg m-2 s-1"
```

Comment:
I'm okay with the custom variable name arg, but this if-clause for precip feels more appropriate in the testing function for now.